docs/source/config/config.rst

```diff
@@ -27,6 +27,20 @@ In :github_st2:`/etc/st2/st2.conf <conf/st2.prod.conf>` include the following se
 
 The ``username`` and ``password`` properties are optional.
 
+StackStorm also supports `MongoDB replica sets <https://docs.mongodb.com/v2.4/core/replication-introduction/>`_
+using `MongoDB URI string <https://docs.mongodb.com/v2.4/reference/connection-string/>`_.
+
+In :github_st2:`/etc/st2/st2.conf <conf/st2.prod.conf>` include the following section :
+
+.. code-block:: bash
+
+    [database]
+    host = mongodb://<#MDB_NODE_1>,<#MDB_NODE_2>,<#MDB_NODE_3>
```

Comment:
Shouldn't this contain replica set name well? "/?replicaSet=<replica-set-name>" . To make sure that the replica set primary is selected.